cspatients/templates/cspatients/patient.html

```diff
@@ -21,10 +21,12 @@ <h4 class="heading heading__subtitle">Personal Details</h4>
             <h5 class="patient-list__label">Surname</h5>
             <h4 class="patient-list__entry">{{ patiententry.surname }}</h4>
           </li>
+          {% if patiententry.age is not None %}
```

Comment:
@DevChima these changes seem contrary to each other.
If `patiententry.age` is `None` then the `default_if_none` filter is not necessary, OR, if it's `None`, we _do_ want to show the AGE, but we want to show that it's blank.

I might be misunderstanding here, please let me know . . .

Reply:
Ah, I see you've answered this in the main thread 😅